packages/dbml-cli/__test__/db2dbml/mssql/expect-out-files/schema.dbml

```diff
@@ -142,7 +142,7 @@ Table "dbo"."StringTypes" {
 
 Table "dbo"."table_with_comments" {
   "created_at" datetime2 [default: `getdate()`, note: 'Timestamp when the item was created.']
-  "description" text [note: 'Detailed description of the item.']
+  "description" text [note: '''Item\'s description''']
```

Comment:
Try something like this:
```
note: '''Item\'s description for \'mssql\''''
```

Reply:
Test added in https://github.com/holistics/dbml/pull/639/commits/ef2b37bdef6fd68f805577fac6a02263e85f6d1e